packages/stringsifter.vm/stringsifter.vm.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>stringsifter.vm</id>
+    <version>2.0</version>
+    <description>StringSifter is a machine learning tool that automatically ranks strings based on their relevance for malware analysis.</description>
+    <authors>Philip Tully, Matthew Haigh, Jay Gibble, and Michael Sikorski</authors>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="vcbuildtools.vm" />
+      <dependency id="python3" version="[3.8.0, 3.9.0)" />
```

Comment:
This mess up the path, see https://github.com/mandiant/VM-Packages/issues/337 as it would be good to discuss a general solution to this problem

Reply:
I'd say we remove until it's updated.